.github/workflows/juju_3_6_ci.yaml

```diff
@@ -0,0 +1,53 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+name: Juju 3.6 CI tests
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  schedule:
+    - cron: '53 1 * * *' # Daily at 01:53 UTC
+
+jobs:
+  lint:
+    name: Lint
+    uses: canonical/data-platform-workflows/.github/workflows/lint.yaml@v18.0.0
+
+  build:
+    name: Build charm
+    uses: canonical/data-platform-workflows/.github/workflows/build_charm.yaml@v18.0.0
+    with:
+      cache: true
+
+  integration-test-3-6:
+    strategy:
+      fail-fast: false
+      matrix:
+        architecture:
+          - amd64
+          - arm64
+    name: Integration | 3.6 | ${{ matrix.architecture }}
+    needs:
+      - lint
+      - build
+    uses: canonical/data-platform-workflows/.github/workflows/integration_test_charm.yaml@v18.0.0
+    with:
+      artifact-prefix: ${{ needs.build.outputs.artifact-prefix }}
+      architecture: ${{ matrix.architecture }}
+      cloud: lxd
+      juju-snap-channel: 3.6/beta
+      libjuju-version-constraint: 3.5.2.0
+      lxd-snap-channel: 5.21/stable
```

Comment:
IMHO, we should use here LXD latest/stable: v.6.1.
Reasons: Field will deploy Juju LTS with latest stable LXD.

Also, please pin MicroK8s here with renovate refresh label as discussed in MM.

Reply:
will add a renovate refresh label for lxd in a separate PR once a preset for lxd has been created.

do we want to use `latest/stable: v6.1` for all juju versions?